tests/lib/util.test.ts

```diff
@@ -723,4 +724,70 @@ describe('Utils', () => {
 			resetMockRandom();
 		});
 	});
+
+	describe('getDisplayAvatar', () => {
+		test('GIVEN user without avatar THEN returns base avatar', () => {
+			const user: APIUser = {
+				discriminator: '0001',
+				avatar: null
+			} as APIUser;
```

Comment:
Is the usage of those bad stubs as opposed to using the mocking tools we have, intended?

https://github.com/skyra-project/skyra/blob/1f8bc3d0babacf1b2e2514ce9023d3642e2e77db/tests/mocks/MockInstances.ts#L9-L20

Reply:
Same, of course, with the rest of the stuff.